lib/core_extensions.rb

```diff
@@ -140,17 +140,20 @@ def to_translation
   end
 
   def to_gb
-    value, _, unit=self.match(/(\d+(\.\d+)?) ?(([KMGT]i?B?|B))$/i)[1..3]
```

Comment:
this used to fail, #match return nil if there's no match and we called `[1..3]` on it